stage2/01-sys-tweaks/files/passwd

```diff
@@ -0,0 +1 @@
+���e�;��
```

Comment:
does this work on every build?
does this depend on the system unique/random salt that gets generated on the first boot?
tbh: i am bit a vague on this;

Reply:
this works on every build. I compared two values from different builds an they are completely the same